server/command_list.go

```diff
@@ -40,7 +40,8 @@ func (p *Plugin) getUpdatedInstallsForUser(userID string) ([]*Installation, erro
 			return nil, errors.Wrapf(err, "could not get updated installation %s", install.ID)
 		}
 		if updatedInstall == nil {
-			return nil, fmt.Errorf("could not find installation %s", install.ID)
+			p.API.LogError(fmt.Sprintf("could not find installation %s", install.ID))
+			continue
```

Comment:
I agree that not giving up entirely here is a good idea, but we should clean up the Mattermost KV store. If, for whatever reason, an installation was deleted from the provisioning server directly we should check for that. It is possible to ask the provisioning server for deleted installations, so we could do that here and then clean up the Mattermost DB. I would think the flow would be:

1. Can't find installation, so ask if deleted or not
2. If deleted, remove from Mattermost DB and send user a message
3. If not deleted, and never was an installation (what happened?!?!) then return error to talk to cloud team.
4. If can't connect to DB them return an error immediately.

Reply:
So, we might want to talk about this, because my understanding is that this is still the flow, with one modification. For #3, instead of returning an error, it merely logs the error, because there may be more than one, so with my change you get all of the installations in state 3 printed as error logs and some output to the user. If we return err, the user gets an error message and only the first error goes to the logs. 

It's possible there's something about this that I am misunderstanding. I have a few orphaned installs in my database currently that cause me to hit the error on this line, so it's possible that I have not successfully reasoned about why I am hitting that failure case.

Finally, this change is isolated from the rest of the PR so if there's something I am not fully understanding here I'm happy to simply remove this from the PR.

Reply:
Re: our discussion, I have opened https://mattermost.atlassian.net/browse/MM-20321 to track the necessary changes to this behavior